pages/index.vue

```diff
@@ -1,5 +1,17 @@
 <template>
   <main class="landing-page">
+    <QiskitBanner padding-x-none class="qiskit-runtime">
+      <div class="bx--grid">
+        <span class="banner-title">Start building with Qiskit runtime.</span>
```

Comment:
@vabarbosa - I forgot to mention, I think the code style used in this project calls for these to be double-underscored, when separating block from element. 

So:
`banner__title`
`banner__message`

then also, the corresponding scss classes below will need to be updated 👇

Reply:
@techtolentino thanks! resolved